mitodl/micromasters
# Send one email per recipient

#### What are the relevant tickets?
closes #981 

#### What's this PR do?
Sends emails to the single users of a search result. The "to"  field of the email contains the actual email of the user.
This PR also introduces a batch send of the emails.

#### Where should the reviewer start?
`mail/api.py`

#### How should this be manually tested?
The case with the overriden email should behave in set same way as before, but you should be able to see the override email in the "to" field.

If there is no `MAILGUN_RECIPIENT_OVERRIDE`, then you should be able to receive one email per user with the user's email in the `to` field.
